docs/resources/iotda_dataforwarding_rule.md

```diff
@@ -137,17 +133,13 @@ The `dis_forwarding` block supports:
 
 * `region` - (Required, String) Specifies the region to which the DIS stream belongs.
 
-* `project_id` - (Required, String) Specifies the project ID to which the DIS stream belongs.
```

Comment:
suggest to make is optional instead of removing it, in case the user is using a subproject

Reply:
ok